Setup/ericomshield-setup.sh

```diff
@@ -2,7 +2,7 @@
 ############################################
 #####   Ericom Shield Installer        #####
 #######################################BH###
-ES_SETUP_VER="Setup:18.08-3007"
+ES_SETUP_VER="Setup:18.08-2608"
```

Comment:
Why version is going down?

Reply:
it's a date 26/08 (today)
it is just a way to see if you have the latest setup file

Reply:
A...